README.md

```diff
@@ -30,136 +23,149 @@ beehive-flow uses the following branch names, each forming part of the process:
  - feature/FEATURE_CODE
  - hotfix/FEATURE_CODE
  - spike/FEATURE_CODE
- 
-Note, there is no `develop` branch and `main` is used instead of `master` as it is GitHub's new default. 
 
-Branch names are enforced and beehive-flow will fail if it encounters other branch names. 
+Branch names and versions are strict and enforced.
 
-### main branch
+All new work happens in `feature/*` branches which are merged to the `main` branch:
 
-A single mainline branch called `main` is used. All features and fixes are merged first to the main branch (via feature branches).
+```
+                       feature/BLAH-123
+  main                 +-----------
+ (x.y.0-alpha)         |           \
+  +--------------------+------------+-----
+```
 
-If features need to be backported to older versions, commits should be cherry-picked to hotfix branches then merged to
-release branches - more on this below.
+The `main` branch is branched to `release/x.y` branches to stabilise a minor version. This is done with `beehive-flow prepare`:
 
 ```
-  main
- (x.y.0-alpha)
-  +---------+----------------------------------+------------------+
-            |                                  |
-            |                                  |
-            |                                  |
-            +-----------------+                +------------------+
-
-             release/1.2                         release/1.3
-             (1.2.x-rc / 1.2.x)                  (1.3.x-rc / 1.3.x)
+  main                 
+ (1.2.0-rc)           (1.3.0-rc)
+  +-----------------+--------------------                             
+         [prepare]: |                             
+                    +--------------------
+                    release/1.2                  
+                    (1.2.0-rc)        
 ```
 
-### release branches
+To make changes to a `release/x.y` branch, create a `hotfix/*` branch, cherry-pick from `main`, then merge:
 
-Release branches are named `release/x.y` where `x.y` is the major.minor version. These are branched off the `main` branch at the beginning of
-*release preparation*. The release branch code is stabilised and then released. A release branch alternates between a "release candidate" state
-and a "release ready" state - more on this below.
+```
+  main
+ (1.2.0-rc)
+  +---------+---------------------------------
+            |              | |  cherry-pick and tweak
+            |              ↓ ↓
+            |            +---------+  hotfix/BLAH-123
+            |            |          \
+            +------------+-----------+--------
+            release/1.3
+            (1.2.0-rc)
+```
 
-### feature branches
+The `main` and `release/x.y` branches can either be in a "release candidate" state with a version like `x.y.z-rc`, or
+a "release ready" state with a version like `x.y.z-rc`. This affects what version is published from CI.
```

Comment:
Are your "release candidate" and "release ready" example versions meant to be the same? I suspect a typo.

Reply:
Oops. Fixed.